tools/noise/gen-profile

```diff
@@ -504,7 +504,7 @@ if [ "$pdf_enabled" ]; then
 		done
 	done
 	echo Filelist: $flist
-	pdftk $flist cat output $resultpdf
+	pdfcat $resultpdf $flist
```

Comment:
So in `tools/noise/subr.sh`, if `pdftk` is installed, then `pdfcat` bash function will call it, else it will call `gs`?